core/drivers/crypto/caam/hal/common/hal_cfg.c

```diff
@@ -31,19 +31,14 @@ enum caam_status caam_hal_cfg_get_conf(struct caam_jrcfg *jrcfg)
 	if (!ctrl_base) {
 		ctrl_base = (vaddr_t)phys_to_virt(CAAM_BASE, MEM_AREA_IO_SEC);
```

Comment:
Could do `core_mmu_add_mapping()` hear  and remove some of the code below instead.